system/STM32WBxx/system_stm32wbxx.c

```diff
@@ -159,7 +159,7 @@
   const uint32_t MSIRangeTable[16UL] = {100000UL, 200000UL, 400000UL, 800000UL, 1000000UL, 2000000UL, \
                                       4000000UL, 8000000UL, 16000000UL, 24000000UL, 32000000UL, 48000000UL, 0UL, 0UL, 0UL, 0UL}; /* 0UL values are incorrect cases */
 
-#if defined(STM32WB55xx) || defined(STM32WB5Mxx) || defined(STM32WB35xx) || defined (STM32WB15xx)
+#if defined(STM32WB55xx) || defined(STM32WB5Mxx) || defined(STM32WB35xx) || defined (STM32WB15xx)|| defined (STM32WB1Mxx)
```

Comment:
This is not part of the automatic commit.
Please add this change in a dedicated commit.
Ex:
system(F7): update the system source file

Reply:
I've updated the PR.